.ci/jobs/apm-integration-tests-selector-mbp.yml

```diff
@@ -0,0 +1,45 @@
+---
+- job:
+    name: apm-integration-tests-selector-mbp
+    display-name: APM Integration Test MBP Selector
+    description: APM Integration Test Selector Multi Branch Pipeline
+    project-type: multibranch
+    number-to-keep: '5'
+    days-to-keep: '1'
+    script-path: .ci/integrationTestSelector.grrovy
+    scm:
+    - github:
+        branch-discovery: all
+        discover-pr-forks-strategy: merge-current
+        discover-pr-forks-trust: permission
+        discover-pr-origin: merge-current
+        discover-tags: true
+        repo: apm-integration-testing
+        repo-owner: elastic
+        credentials-id: 2a9602aa-ab9f-4e52-baf3-b71ca88469c7-UserAndToken
+        ssh-checkout:
+          credentials: f6c7695a-671e-4f4f-a331-acdce44ff9ba
+        build-strategies:
+        - tags:
+            ignore-tags-older-than: -1
+            ignore-tags-newer-than: -1
+        - regular-branches: true
```

Comment:
the build will be trigger on changes in the branches, but it is not prepared for that, isn't?